bnding_box_yolo_format.py

```diff
@@ -0,0 +1,110 @@
+import os, json, requests
+from PIL import Image
+from io import BytesIO
+import pandas as pd
+import numpy as np
+from sys import argv
+
+def get_box_centers_all_emotions(sample_row):
+    '''
+    input: A dictionary of the format {emotion1: [{geometry_box1: [list of 4 corner coordinates],
+                                                   geometry_box1: [list of 4 corner coordinates],
+                                                   ...}]
+                                       emotion2: [...],
+                                       ...
+                                       emotionN: [...]
+                                       }
+    output: A dictionary of the format {emotion1: [(center1 coordinates), (center1 coordinates), ...],
+                                        emotion2: [...],
+                                        ...
+                                        emotionN: [...]
+                                       }
+    '''
+
+    lab = json.loads(sample_row)
+    centers = {}
+    for key, val in lab.items():
+        lab_df = pd.DataFrame(val)
+        centers[key] = lab_df.iloc[:, 0].apply(lambda s: get_box_center(s)).tolist()
```

Comment:
smart! but hard to understand. perhaps provide some example. like what key actually is...